uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsenceType.php

```diff
@@ -197,7 +197,7 @@ private static function unsetDefaultTypes() {
    */
   private static function validateParams($params) {
     if(!empty($params['add_public_holiday_to_entitlement'])) {
-      self::validateAddPublicHolidayToEntitlement();
+      self::validateAddPublicHolidayToEntitlement($params);
```

Comment:
I'm passing the $params instead of only the ID to keep it consistent with the other validation methods that also expect the array. There's also the little advantage that, if we need to use more fields in the validation, they'll already be there.